### ISSUE TITLE
How to enable the top right code block copy button?

### ISSUE THREAD

**Issue by OpenJarvisAI** [asjqkkkk/markdown_widget#130]
Maybe even customize it?

**Comment by asjqkkkk** [comment]
Yes, you need to customize it, such as https://github.com/asjqkkkk/markdown_widget/blob/master/example/lib/widget/code_wrapper.dart

**Comment by OpenJarvisAI** [comment]
hello ,

PreConfig(
          // theme: myDarkTheme,
          // language: 'dart',
          textStyle: const TextStyle(fontSize: 14),
          wrapper: wrapper,
        ),

but this theme can not switch any longer

**Comment by asjqkkkk** [comment]
Hi @OpenJarvisAI , you can change the theme by passing different `PreConfig`

**Comment by gnodabb** [comment]
I apologize up front. I'm very new to flutter/dart.
I just want a simple code copy button. How do you define `wrapper`, when you pass into the config?
Do you have to parse the code blocks out manually to achieve this functionality?
The example shows the code copy wrapper. But not how to use it in the PreConfig.

**Comment by asjqkkkk** [comment]
here is the example for adding a copy button
https://github.com/asjqkkkk/markdown_widget/blob/56131aae16f514b243d87aa4c9b55ceb9f8aec58/example/lib/pages/markdown_page.dart#L99-L107
